addons/event/models/ir_autovacuum.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models, fields
+
+
+class AutoVacuum(models.AbstractModel):
+    _inherit = 'ir.autovacuum'
+
+    @api.model
+    def power_on(self, *args, **kwargs):
```

Comment:
Why is this done by the vacuum cleaner and not a dedicated cron job ?!

Reply:
@Julien00859 to avoid to create again a new CRON

Reply:
What is the real question @Julien00859  ?

Reply:
Task 2154079, changing inheritance of ir.autovacuum to a `_autovacuum` class model attribute listing the garbage collecting functions. Merged this one and was confuse it was not "vacuuming" anything (=unlinking/removing stuff).

Reply:
Ok. Indeed it is not a real vacuum, but as we try to reduce number of different crons that do not require a dedicated entry, we generally override the vacuum cleaner for small job. If you think we should change it don't hesitate.

Reply:
Ok. The best would be to have a dedicated "misc" cron job with a very slow frequency which is kinda what the autovacuum is. Let's just leave it that way, it's fine, thank you both :)